source/intro.tex

```diff
@@ -600,6 +600,10 @@
 \Cpp{} and C\@. Certain features of \Cpp{} exist solely for compatibility
 purposes; \ref{depr} describes those features.
 
+\pnum
+Throughout this document,
+non-normative encouragement for implementations is introduced by \recommended.
```

Comment:
That sounds pretty definitive.  Are you implying *all* instances of non-normative encouragement? What about all the places we missed, and are still in notes or wherever?